Lib/test/test_http_cookies.py

```diff
@@ -58,6 +59,43 @@ def test_basic(self):
             for k, v in sorted(case['dict'].items()):
                 self.assertEqual(C[k].value, v)
 
+    def test_unquote(self):
+        cases = [
+            (r'a="b=\""', 'b="'),
+            (r'a="b=\\"', 'b=\\'),
+            (r'a="b=\="', 'b=\\='),
+            (r'a="b=\n"', 'b=\\n'),
```

Comment:
This PR only unquotes `\\` and `\"` which are produced by `_quote()`. The current code removes backslash in all cases. I do not know what behavior is correct (maybe `\n` should be interpreted as a newline character?).

Reply:
(For other reviewers, current behavior is maintained with https://github.com/python/cpython/pull/123075/commits/ab87c992c2d4cd28560178048915bc9636d6566e)